### ISSUE TITLE
Facebook Comments Lightbox: JS Error in IE 8

### ISSUE THREAD

**Issue by kartzke** [myApi/myApi#12]
When using IE8, I get the following JavaScript error when I click the "Add a Comment" lightbox link:

Webpage error details

User Agent: Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; .NET CLR 1.1.4322; .NET CLR 2.0.50727; .NET CLR 3.0.4506.2152; .NET CLR 3.5.30729; .NET4.0C; MS-RTC LM 8)
Timestamp: Mon, 18 Apr 2011 19:41:31 UTC

Message: Object doesn't support this property or method
Line: 30
Char: 5
Code: 0
URI: /plugins/system/myApiModal/myApiModal.js

The lightbox does not pop up for me when using Internet Explorer, but it does work with Firefox 4 and Chrome.

**Comment by thomaswelton** [comment]
On this page I have the comment opening in a lightbox
http://www.myapi.co.uk/index.php?option=com_content&view=frontpage&Itemid=61

But when I use IE8 I get no error messages. Could you provide a link to your site and also confirmation that the error still occurs.

**Comment by kartzke** [comment]
I got a new computer which uses IE 9 now and I do not have the option to downgrade back to IE 8 (work computer).  

Using the link you sent me, IE 9 does not get any JavaScript errors. I manually disabled the "Add a Comment" code, so it is not live on a site right now.  I'll try to re-enable this in the next day or so and report back.

**Comment by thomaswelton** [comment]
Using IE 9's developer tools (press F12) I was able to change to document mode to IE 8, and did not get the errors. It would not surprise me if there was an IE specific issue here, but I may have fixed it on my site with a newer release, or maybe it specific to your setup. Keep me posted.

**Comment by kartzke** [comment]
I just installed the latest version of myAPI (1.3.1-62) to my testing site. When I switched to IE 8 doc mode and I get the following script error when I click the Add Comment link from the Frontpage:

SCRIPT438: Object doesn't support property or method 'getStyle' 
myApiModal.js, line 30 character 5

I've disabled all GZipping and compression plugins. I've also disabled a wide variety of other plugins that might use javascript and cause a conflict, but the problem persists in IE 8.  

Using IE 9 doc type and Safari 5.0.5, the Lightbox pops up, but the comments don't load.  Instead, I get the "Sorry, something went wrong" facebook error.

The lightbox and comments seem to load fine in Firefox 4 and Chrome 11.0.696.68.
